devenv/local/docker-compose/README.md

```diff
@@ -0,0 +1,54 @@
+# DOCKER COMPOSE INSTRUCTIONS
+
+
+## [1] Build all containers:
+
+##### `sh build.sh`
+
+## [2] Spin up all containers:
+
+##### `sh up.sh`
```

Comment:
Should we consider adding top-level make targets for spinning up the local dev env? It would be nice to be able to run `make local-dev-env` from the project root. It can definitely be done as a follow-up.